mongo-helm/templates/mongo-express.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongo-express
+  labels:
+    app: {{ .Values.mongoexpress.appName }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Values.mongoexpress.appName }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.mongoexpress.appName }}
+    spec:
+      containers:
+      - name: {{ .Values.mongoexpress.appName }}
+        image: {{ .Values.mongoexpress.imageName }}:{{ .Values.mongoexpress.imageTag }}
+        ports:
+        - containerPort: 8081
+        env:
+        - name: ME_CONFIG_MONGODB_ADMINUSERNAME
+          valueFrom:
+            secretKeyRef:
+              name: mongodb-secret
+              key: mongo-root-username
+        - name: ME_CONFIG_MONGODB_ADMINPASSWORD
+          valueFrom: 
+            secretKeyRef:
+              name: mongodb-secret
+              key: mongo-root-password
+        - name: ME_CONFIG_MONGODB_SERVER
+          valueFrom: 
+            configMapKeyRef:
+              name: mongodb-configmap
+              key: database_url

```

Comment:
Please make it so I am able to add more environment variables if need be.

Reply:
Done.